File: 00_Learn_The_Basics/Things_To_Know_In_A_Language/For_Loops/Table.java
// Printing a multiplication table 

import java.util.*;

public class Table
{
    public static void main(String[] args)
     {
        
    int n,m=1;
    Scanner in=new Scanner(System.in);

    System.out.println("Enter the numberwhose table you want ");
    n=in.nextInt();
    
    for(int i=1;i<=10;i++)
    {
        m=n*i;
        System.out.println(n+ " X "+i+ "= "+ m);
    }
    
}
}

File: 00_Learn_The_Basics/Things_To_Know_In_A_Language/For_Loops/table.cpp
// Printing a multiplication table 

#include<iostream>
using namespace std;

int main()
{
    int n,m=1;
    cout<<"Enter the numberwhose table you want "<<endl;
    cin>>n;
    
    for(int i=1;i<=10;i++)
    {
        m=n*i;
        cout<<n << " X "<<i<< "= "<< m <<endl;
    }
    
}
